### ISSUE TITLE
question ?

### ISSUE THREAD

**Issue by matrixreal** [OgarProject/Ogar#360]
Hi,
i'm searching about the line wich is related to the cell devide when it's big and colloide with the virus ?
thank you

**Comment by Ogarunite** [comment]
I read the question as "Question, Questionmark"
So the question would be questioning the questionmark whether the question itself is a questionmark, or the questionmark fullfills the purpose of the string "question":

On topic: Search for "Newcellvirused" in gameserver.js
The new cell(s) created is the var NewCell
If you want to change how many cells get divided, etc, search inside the virus.js in the entity folder.

**Comment by matrixreal** [comment]
i want to add a sound effect when the cell (player) is big and get devided when it colloide with the virus

and i like you philosophic words ... it took me 5 minutes to understand :dancer:

**Comment by MastaCoder** [comment]
You can't do that. Sound can be transmitted only on the client, not on the server. Only if you want to take the long path and get the server and client to work together.

**Comment by matrixreal** [comment]
@Ogarunite 
yes ... i would like to add a sound when the cell is divided but can't find it
